deployment/docker-compose/README.md

```diff
@@ -0,0 +1,162 @@
+# Docker Compose Deployment
+
+The CCNP can be deployed in the TDVM by docker compose.
+
+## 1. Prerequisite
+
+### 1.1 Host Side 
+
+The host should enable TDX, and setup service PCCS & service QGS which help generate TDX quote in the guest.
```

Comment:
which help to generate TDX quote in the guest

Reply:
Sure! Append a reference to the white paper involving in all dependencies.